src/Feed.jsx

```diff
@@ -0,0 +1,433 @@
+                  popupState.close();
+                }}>Amount</MenuItem>
+            <MenuItem                 onClick={() => {
+                  setSortField('description');
+                  popupState.close();
+                }}>Description</MenuItem>
+          </Menu>
+        </React.Fragment>
+      )}
+    </PopupState>
+</div>
+<div style={{ marginRight: '10px' }}>
+    <PopupState variant="popover" popupId="demo-popup-menu">
+      {(popupState) => (
+        <React.Fragment>
+          <Button variant="contained" {...bindTrigger(popupState)}>
+          Sort Order
+          </Button>
+          <Menu {...bindMenu(popupState)}>
+            <MenuItem                 onClick={() => {
+                  setSortOrder('asc');
+                  popupState.close();
+                }}>Ascending</MenuItem>
+            <MenuItem                 onClick={() => {
+                  setSortOrder('desc');
+                  popupState.close();
+                }}>Descending</MenuItem>
+          </Menu>
+        </React.Fragment>
+      )}
+    </PopupState>
+    </div>
+    </div>
+<Button onClick={handleSort}></Button>
+    </Box>
+    <div style={{ display: 'flex', flexWrap: 'wrap', justifyContent: 'space-between' }}>
+      {transactionsToDisplay.map((trans, index) => (
+                 <Card
+                 key={index}
+                 sx={{
+                   flexBasis: 'calc(33.33% - 10px)', 
+                   margin: '5px',
+                   width: '100%', 
+                 }}
+               >
+          <CardContent>
+            <Typography variant="h6" color="blue">
+              Title: {trans.title}
+            </Typography>
+            <Typography variant="h6" color="red">
+              Date:{trans.date}
+            </Typography>
+            <Typography variant="h6" color="text.secondary">
+              Category:{trans.category}
+            </Typography>
+            <Typography variant="h6" color="text.primary">
+              Amount:{trans.amount}
+            </Typography>
+            <Typography variant="h6" color="gray">
+              Description:{trans.description}
+            </Typography>
+            <Button onClick={() => handleEditClick(trans)}>Edit</Button>
+            <Button onClick={() => handleDeleteClick(trans.id)}>Delete</Button>
+            <div style={{ position: 'absolute', top: 0, right: 0 }}>
+              <IconButton aria-label="settings" onClick={handleMenuClick}>
+                <MoreVertIcon />
+              </IconButton>
+              <Menu anchorEl={anchorEl} open={Boolean(anchorEl)} onClose={handleCloseMenu}>
+                <MenuItem onClick={handleCloseMenu}>Edit</MenuItem>
+                <MenuItem onClick={handleCloseMenu}>Delete</MenuItem>
+              </Menu>
+            </div>
+          </CardContent>
+        </Card>
+      ))}
+      </div>
+
+     
+      <div style={{ display: 'flex', justifyContent: 'center', marginTop: '20px' }}>
```

Comment:
Pls, use material UI for styling. Ex: here you can use the **Box** component. for custom styling you can look into **material UI-styled components**